src/MudBlazor/Base/MudComponentBase.cs

```diff
@@ -3,35 +3,35 @@
 
 namespace MudBlazor
 {
-    public abstract class MudComponentBase : ComponentBase
+    public abstract class MudIComponent : IComponent
```

Comment:
```suggestion
    public abstract class MudComponentBase : IComponent
```
That's the only line that is necessary. No need to change hundreds of files. Can you please undo everything else?